### ISSUE TITLE
Getting Started Problem

### ISSUE THREAD

**Issue by fanwenzh** [authzforce/core#9]
here i am sorry for this simple initial problem:
when i run core following Getting Started, and enter the commandline "java -jar ./authzforce-ce-core-pdp-cli-10.2.0.jar pdp.xml Request.xml ", it show error command.error like this:
```code
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
	at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:576)
Caused by: picocli.CommandLine$ExecutionException: Error while calling command (org.ow2.authzforce.core.pdp.cli.PdpCommandLineCallable@6ed3ef1)
	at picocli.CommandLine.execute(CommandLine.java:453)
	at picocli.CommandLine.access$300(CommandLine.java:130)
	at picocli.CommandLine$RunLast.handleParseResult(CommandLine.java:533)
	at picocli.CommandLine.parseWithHandlers(CommandLine.java:651)
	at picocli.CommandLine.call(CommandLine.java:878)
	at picocli.CommandLine.call(CommandLine.java:829)
	at org.ow2.authzforce.core.pdp.cli.PdpCommandLineCallable.main(PdpCommandLineCallable.java:150)
	... 8 more
Caused by: java.lang.NullPointerException
	at org.ow2.authzforce.core.pdp.impl.PdpEngineConfiguration.getInstance(PdpEngineConfiguration.java:516)
	at org.ow2.authzforce.core.pdp.impl.PdpEngineConfiguration.getInstance(PdpEngineConfiguration.java:634)
	at org.ow2.authzforce.core.pdp.cli.PdpCommandLineCallable.call(PdpCommandLineCallable.java:92)
	at org.ow2.authzforce.core.pdp.cli.PdpCommandLineCallable.call(PdpCommandLineCallable.java:60)
	at picocli.CommandLine.execute(CommandLine.java:451)
	... 14 more
  ```
picocli.CommandLine$ExecutionException: Error while calling command, is it something wrong with my enviroment?
on the other hand, class StandardEnvironmentAttributeSource in org.ow2.authzforce.core.xmlns.pdp  (pdp-engine mainTest class) did not exist

**Comment by cdanger** [comment]
Indeed, there seems to be some bug with relative paths in last version. We'll have to fix that ASAP. Workaround: use absolute paths for the pdp.xml and request.xml. Also request.xml should be "IIA00fo1/Request.xml" if you followed the Getting Started literally (you copied the whole folder IIA01). Last, the JAR is fully executable so assuming you have permission to execute (do `chmod a+x` on it if necessary), you don't need "java -jar" anymore.  So in my case, this works:

```
$ ./authzforce-ce-core-pdp-cli-10.2.0.jar /my/absolute/path/to/pdp.xml /my/absolute/path/to/IIA001/Request.xml
```

**Comment by fanwenzh** [comment]
Thank you, it is solved!

**Comment by cdanger** [comment]
Hello, we just released new version 10.3.0 with the fix, so you can use relative paths again with this new version. It should be published to Maven central within the hour or so.

As for the question on missing classes, not sure I understand. Do you have the error when running the executable jar (authzforce-ce-core-pdp-cli-XXX.jar) as well? or are you trying to run things from the sources (Maven project) ?

I'll assume this is the second case, because we don't have the issue in the first case: these classes are there in the `.../lib/authzforce-ce-core-pdp-engine-10.3.0.jar` that is embedded in the executable.
In the second case, you are working with a Maven project and these classes (among others) are automatically generated by Maven build (from XML schema `pdp-engine/src/main/resources/pdp.xsd`) during `generate-sources` phase (look at the POM for more info). So you have to do at least a `mvn clean generate-sources` command (or similar with your favorite IDE) to get them. Some IDE (like recent Eclipse version) may generate it automatically by looking at the POM, but I guess it is not your case.
If you need more help, please give the full stacktrace of the error to better understand the context.
